collector/test/NetworkStatusNotifierTest.cpp

```diff
@@ -64,15 +65,6 @@ class Semaphore {
   std::mutex mutex_;
 };
 
-class MockCollectorConfig : public collector::CollectorConfig {
- public:
-  MockCollectorConfig() : collector::CollectorConfig(0) {}
-
-  void DisableAfterglow() {
-    enable_afterglow_ = false;
-  }
-};
-
```

Comment:
This is not needed anymore as afterglow is disabled using an environment variable.

Reply:
I am not using a mock instead of an environment variable, as it was originally.